src/components/TimeCardPage/SubmitCard.tsx

```diff
@@ -8,39 +8,57 @@ import Button from 'react-bootstrap/Button';
 
 export default function SubmitCard(props) {
```

Comment:
Now that we have Typescript, we should also define what props exist. For example (not necessarily with the right types):

```ts
interface SubmitCardProps {
  readonly comment: string;
  readonly content: string;
}

export default const SubmitCard: React.FC<SubmitCardProps> = ({
  comment,
  content
}) => {
```

Note how we can also destructure the props so that later in the component, we can just write `content` instead of `props.content`